packages/server/api/src/app/ai/mcp/docs-tools.ts

```diff
@@ -1,31 +1,56 @@
+import { experimental_createMCPClient as createMCPClient } from '@ai-sdk/mcp';
 import { AppSystemProp, logger, system } from '@openops/server-shared';
-import { experimental_createMCPClient as createMCPClient, tool } from 'ai';
-import { Experimental_StdioMCPTransport as StdioMCPTransport } from 'ai/mcp-stdio';
+import { tool } from 'ai';
 import { z } from 'zod';
 import { MCPTool } from './types';
 
+type MCPSearchTool = {
+  execute: (
+    args: { query: string },
+    options: { toolCallId: string; messages: unknown[] },
+  ) => Promise<unknown>;
+};
+
+type MCPHttpTransport = {
+  type: 'http';
+  url: string;
+};
+
 export async function getDocsTools(): Promise<MCPTool> {
-  const mcpServerPath = system.get<string>(AppSystemProp.DOCS_MCP_SERVER_PATH);
-  if (!mcpServerPath) {
+  const mcpServerUrl = system.get<string>(AppSystemProp.DOCS_MCP_SERVER_PATH);
+  if (!mcpServerUrl) {
     return {
       client: undefined,
       toolSet: {},
     };
   }
 
   logger.debug('Creating MCP client for docs', {
-    serverPath: mcpServerPath,
+    serverPath: mcpServerUrl,
   });
 
   const client = await createMCPClient({
-    transport: new StdioMCPTransport({
-      command: 'node',
-      args: [mcpServerPath],
-    }),
+    transport: {
+      type: 'http',
+      url: mcpServerUrl,
+    } as MCPHttpTransport,
   });
 
   const tools = await client.tools();
-  const searchTool = tools['search'];
+  const toolsObject = tools instanceof Map ? Object.fromEntries(tools) : tools;
```

Comment:
the changes here are needed because the new package could return it as a map, so we cast it to an object